examples/demo-simple-stf/README.md

```diff
@@ -68,11 +68,11 @@ Now that we have defined the necessary types, we need to implement the following
 
 These functions handle the initialization and preparation stages of our rollup, but as we are not modifying the rollup state, their implementation is simply left empty.
 
-Next we need to write the core logic in `apply_blob`:
+Next we need to write the core logic in `apply_tx_blob`:
 
 ```rust
     // The core logic of our rollup.
-    fn apply_blob(
+    fn apply_tx_blob(
```

Comment:
Why is this renamed?

Reply:
That's a typo: this is going to be renamed when the optimistic module lands, but it seems I propagated it too soon enough!

Reply:
Can we bring it back for now then?